src/android/Print.java

```diff
@@ -259,6 +259,31 @@ private boolean createReceiptData(Order order) {
 			if (order.customer_address != null && !order.customer_address.isEmpty())
 				builder.append("Address: " + order.customer_address + "\n");
 
+			if (order.handoff_provider != null && !order.handoff_provider.isEmpty()) {
+				String formatted_provider = "";
+
+				switch (order.handoff_provider) {
+					case "ubereats": formatted_provider = "Uber Eats";
+					break;
+
+					case "postmastes": formatted_provider = "Postmates";
+
+					case "doordash": formatted_provider = "DoorDash";
+					break;
+
+					case "grubhub": formatted_provider = "Grubhub";
+					break;
+
+					case "skip": formatted_provider = "Skip the Dishes";
+					break;
+				}
```

Comment:
This will require a change to the plugin anytime we add a new provider -- can we send down the formatted name from the API side instead?